src/agent0/ethpy/base/transactions.py

```diff
@@ -382,6 +384,7 @@ def build_transaction(
     nonce: Nonce | None = None,
     read_retry_count: int | None = None,
     txn_options_value: int | None = None,
+    txn_options_gas: int | None = None,
```

Comment:
are we still using this? Is this the arg that the policy config is setting?

Reply:
yeah this is the lowest-level function that has that policy parameter passed to it, then we add it to transaction kwargs here